SA/design_concept/content/design/flo_inspired_design.typ

```diff
@@ -1,59 +1,59 @@
 With this design, function elements are distributed on a canvas.
-Every element (e.g. a variable or another function) can be connected to another element with arrows.
+Every element (e.g., a variable or function) can be connected to another element with arrows.
 
 == Function Parameter Editor
-Parameters of a function are defined seperately from the function body.
-When opening a function with the editor, a dialog, as depicted in @flo_inspired_param_editor, appears next to the editor canvas.
+Parameters of a function are defined separately from the function body.
+A dialog, as depicted in @flo_inspired_param_editor, appears next to the editor canvas when opening a function with the editor.
 #figure(
     image("../../static/flo_inspired_parameter_editor.png", width: 40%),
     caption: "Draft of proposed function parameter editor"
 )<flo_inspired_param_editor>
-By clicking on the plus sign, a user can add a parameter and name it. By clicking on the minus sign, a parameter can be removed again.
-To use a prameter in the function body, the user can simply drag it from the prameter editor onto the function editor canvas.
+Users can add and name a parameter by clicking the plus sign. By clicking on the minus sign, a parameter can be removed again.
+The user can drag a parameter from the parameter editor onto the function editor canvas to use a parameter in the function body.
 
 == Function Editor
-To define a function, the user can drag pre-defined functions and block elements as well as self-defined functions from a sidebar onto the editor canvas.
+To define a function, the user can drag pre-defined functions, block elements, and self-defined functions from a sidebar onto the editor canvas.
 As described above, the same can be done with function parameters from the parameter editor.
-To connect a parameter to a function call, the user can create a connector-arrow berween them.
-This can be achieved by Ctrl+Click on the parameter and then click on the function parameter slot.
+To connect a parameter to a function call, the user can create a connector-arrow between them.
+This is achieved by clicking on the parameter while holding Ctrl and then clicking on the function parameter slot.
 
-To visually display currying, every function block is only allowed to have one or no parameter.
-If a function has more than one parameter, the function block has depending blocks for every additional parameter.
-Such depending blocks can be recognized by the ":apply" suffix to the function's name.
-The last block of a function is the value return by the function.
-This value can be used as a parameter for another function or marked as the return value of the function.
+To visually display currying, every function block has only one or no parameter.
+If a function has more than one parameter, the function block has additional blocks for every additional parameter.
```

Comment:
```suggestion
If a function has more than one parameter, the function block has dependent blocks for every additional parameter.
```
I think it's important here to make it clear that these block depend on the previous one.